tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -5,8 +5,8 @@
 
 from app.models import (
     Notification, ScheduledNotification, SCHEDULE_NOTIFICATIONS,
-    EMAIL_TYPE, INTERNATIONAL_SMS_TYPE, SMS_TYPE
-)
+    EMAIL_TYPE, INTERNATIONAL_SMS_TYPE, SMS_TYPE,
+    NotificationEmailReplyTo)
```

Comment:
Reorganise this to make it clearer, so group models together and enums together

Reply:
done